test/parallel/test-buffer-sharedarraybuffer.js

```diff
@@ -3,8 +3,6 @@
 // Flags: --harmony-sharedarraybuffer
 
 require('../common');
-const assert = require('assert');
```

Comment:
@lenakaplan Could you also undo this `assert` deletion so we can run a new CI on this PR?